editor/sidebar/featured-image/index.js

```diff
@@ -135,4 +79,19 @@ export default connect(
 			},
 		};
 	}
+);
+
+const fetchAPIData = withAPIData( ( { featuredImageId } ) => {
```

Comment:
Do we want to use the naming convention discussed at https://github.com/WordPress/gutenberg/pull/2525#discussion_r135025408 ?

Reply:
Sure, I was waiting for your changes there :). Updating